controller/static/app/containers/deploy.controller.js

```diff
@@ -8,23 +8,44 @@
     ContainerDeployController.$inject = ['$http', '$state'];
     function ContainerDeployController($http, $state) {
         var vm = this;
-        vm.deploy = deploy;
+        vm.cmd = "";
+        vm.envVars = [];
+        vm.variableName = "";
+        vm.variableValue = "";
         vm.deploying = false;
         vm.containerName = "";
         vm.error = "";
         vm.request = {
+            Env: [],
             AttachStdin: false,
             Tty: true,
         };
 
+        vm.deploy = deploy;
+        vm.pushEnvVar = pushEnvVar;
+
         ////
```

Comment:
I've just seen that John Papa's code was using them to separate bindable members from non-bindable.

The idea being that all the bindable members should be listed up top, so that they can be seen 'at a glance'.  Bindable functions are assigned to a private function that is defined below.

Not too fussed about the separator either, so I've removed them!

Cheers :smile: